script/lib/omaha.py

```diff
@@ -71,7 +71,7 @@ def get_base64_authorization(omahaid, omahapw):
 
 # To-Do: Add functions to create apps
 def get_appguid(channel, platform):
-    if channel in 'dev' or channel in 'release' and platform in 'darwin':
+    if channel in 'dev' or platform in 'darwin' and channel in ['beta', 'dev', 'release']:
```

Comment:
I think maybe you mean:
`if channel in ['dev'] or platform in ['darwin'] and channel in ['beta', 'dev', 'release']:`


Otherwise for example 'e' is in 'dev'

Reply:
or by using `==` like `channel == 'dev'`

Reply:
I guess I can't resolve a change request by you @bbondy. Do you mind resolving it so I can merge?